data/pages.yaml

```diff
@@ -53,6 +49,56 @@ novice:
   learn: Understand the fundamentals of Ansible automation
   install: Install the Ansible package
   start: Write your first playbook in a few easy steps
+# Resources for user journeys
+user:
+  title: User resources
+  intro: Automate the management of remote systems and control their desired state.
+  milestone:
+    automate: Create automation
+    build_inventory: Build inventories
+    organize: Organize automation projects
+    tooling: Use Ansible tooling
+    reuse: Re-use automation content
+    share: Share automation content
+    awx: Schedule and run automation jobs
+    executor: Embed automation in systems
+  automate:
+    intro: Write playbooks and develop content that automates complex tasks.
+    start: Start writing Ansible playbooks
+    evaluate_plugins: Learn which plugins to use for your tasks
+  build_inventory:
+    intro: Build inventories that list and describe your infrastructure.
+    multiple_hosts: Build inventory files to manage multiple hosts
+    dynamic: Use dynamic inventories
+  organize:
+    intro: Organize your automation project as it grows.
+    roles: Use roles to structure the automation project
+  tooling:
+    intro: Validate and improve your project for consistent, trusted automation.
+    lint: Use Ansible Lint to validate playbooks
+    molecule: Install Molecule to develop and test Ansible roles
+    navigator: Create and test playbooks with Ansible Navigator
+    vscode: Use Ansible with Visual Studio Code and OpenVSX compatible editors
+  reuse:
+    intro: Re-use existing automation content.
+    explore_galaxy: Start exploring Ansible Galaxy
+    install_roles: Install and use roles
+    install_collections: Install and use collections
+    customize_content: Customize automation content
+  share:
+    intro: Share your automation content with others.
+    submit_roles: Submit roles to an existing collection
+    create_collection: Create a new collection
+    upload_collection: Upload a collection to Ansible Galaxy
+  awx:
+    intro: Use AWX for more convenient automation.
+    on_demand_jobs: Execute automation jobs on demand
+    schedule_jobs: Schedule automation jobs
+    create_job_templates: Create AWX job templates
+  executor:
+    intro: Invoke automation jobs from other systems.
```

Comment:
Is this the real purpose of EEs? I'd definitely change jobs as that seems an AWX term.  Are EEs more like portable containers for select versions of core, collections, and ..other bits... which of course does not make for a snappy intro...

Reply:
I'm no expert here but runner and sdk provide a programmatic interface to ansible so you can invoke automation runs from different systems or, say, a python application. runner/sdk call into to ansible runtime execution.

EEs give you a way to isolate dependencies for control nodes. e.g. you want to update databases you could use an ee with postgres drivers. @Spredzy could probably help check my understanding and make some clarifications.